web/components/contract/contract-tabs.tsx

```diff
@@ -416,6 +466,76 @@ export const CommentsTabContent = memo(function CommentsTabContent(props: {
   )
 })
 
+const PinnedComment = (props: {
+  contract: Contract
+  comment: ContractComment
+  seeReplies: boolean
+  numReplies: number
+  trackingLocation: string
+  bets?: Bet[]
+  pinnedHighlighted?: boolean | undefined
+  setIdInUrl: React.Dispatch<React.SetStateAction<string | undefined>>
+}) => {
+  const {
+    comment,
+    contract,
+    trackingLocation,
+    bets,
+    seeReplies,
+    numReplies,
+    pinnedHighlighted,
+    setIdInUrl,
+  } = props
+
+  const handleContextClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault()
+    const commentId = comment.id
+    setIdInUrl(commentId)
```

Comment:
Could try deleting this, seems like it should work if we are updating the hash (and listening for hash changes above)

Reply:
WRONG